core/libraries/ifix-services-common/pom.xml

```diff
@@ -1,73 +1,73 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Why are there any changes to this file?

Reply:
haven't done any changes in this file. while taking pull from develop and got conflict and accepted the develop branch one.